physics/Interstitials/UFS_SCM_NEPTUNE/GFS_phys_time_vary.scm.F90

```diff
@@ -16,6 +16,7 @@ module GFS_phys_time_vary
 
       use aerclm_def, only : aerin, aer_pres, ntrcaer, ntrcaerm, iamin, iamax, jamin, jamax
       use aerinterp,  only : read_aerdata, setindxaer, aerinterpol, read_aerdataf
+      use aerinterp,  only : read_aerdata_dl, aerinterpol_dl, read_aerdataf_dl
```

Comment:
Same comment as in GFS_phys_time_vary.fv3.F90

Reply:
change intent(inout) to intent(out) for all related errmsg and errflag and committed